common/src/main/java/com/itsaky/androidide/utils/NavigationBar.kt

```diff
@@ -0,0 +1,85 @@
+/*
+ *  This file is part of AndroidIDE.
+ *
+ *  AndroidIDE is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  AndroidIDE is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *   along with AndroidIDE.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.itsaky.androidide.utils
+
+import android.annotation.SuppressLint
+import android.content.Context
+import androidx.annotation.IntDef
+
+// Source: https://gist.github.com/Thorsten1976/07d61b3f697364e5f1c08ae076641d58
+
+object NavigationBar {
```

Comment:
I added this utility class from one of my personal projects. Initially it was found somewhere on the internet. I used it to create fancy integrations with system bars in landscape mode ([link](https://github.com/Smooth-E/wireless-adb-switch/blob/dcaab5e3da7af3bbac92e63075fffbf40ecb336c/widgetFactory/src/main/java/com/smoothie/widgetFactory/FullScreenActivity.kt)).

Reply:
The gist file doesn't include a license and you have included a link to the gist, so it should be fine.